tests/smoke/base.sh

```diff
@@ -47,6 +47,8 @@ elif command -v flit &> /dev/null; then
   flit install
 elif command -v meson &> /dev/null; then
   pip install -e .
+elif  command -v setuptools &> /dev/null; then
+  pip install --editable .
```

Comment:
@xmnlab This is the same as for meson, do I leave it as it is now or change it to `pip install -e .`?